docs/concepts/tracking_logs.rst

```diff
@@ -0,0 +1,9 @@
+Tracking Logs
+*************
+
+The event-tracking library tracks context-aware semi-structured system events. It captures 
+and stores events with nested data structures in order to truly take advantage of schemaless
+data storage systems.
+
+This library produces trackings logs which can be stored in multiple destinations. Those trackings logs
+are used to generate reports and dashboards.
```

Comment:
Technically we're not using them to generate any reports at this point, but they _can_ be used for that if people want to. We should make it clear that we optionally capture this data for people to use in their own reporting, but don't expose it in any way ourselves (at least not yet). Should also mention that the reason we do this is because there is a lot more data captured in tracking logs than in the xAPI statements, but at the cost of also storing a lot of PII including usernames, email addresses, browser data, and IP addresses.